apps/ex_wire/lib/ex_wire/sync.ex

```diff
@@ -281,23 +284,26 @@ defmodule ExWire.Sync do
           Blocktree.t(),
           Blocktree.t(),
           Trie.t(),
-          BlockQueue.t(),
-          integer()
-        ) :: {integer(), Trie.t()}
+          BlockQueue.t()
+        ) :: Trie.t()
   defp maybe_request_new_block_or_save(
```

Comment:
I know this is not part of the changes, but would it make sense to break this into two functions? Maybe that would help avoid the nested `if` statements?

Reply:
Yup, moved to two functions.